tests/testthat/test-free_cores.R

```diff
@@ -0,0 +1,7 @@
+test_that("free_cores throws deprecation warning", {
+  skip_on_cran()
+  expect_warning(
+    free_cores(),
+    regexp = "`free_cores\\(\\)` was deprecated in mmrm 0\\.1\\.6."
```

Comment:
consider `fixed = TRUE` then you don't need the escapes

Reply:
I am happy with escapes.

Reply:
hm I would prefer without :-)

Reply:
It's definitely more of a fixed string match rather than a regex, isn't it?

Reply:
I just find it weird to use an argument calle "regex" that is not interpeted as a regex but all fine by me, changed it.

Reply:
do a PR to `testthat` for a rename of the argument ;-)

Reply:
@BrianLang haha :-) that will be difficult!